src/darwin/Framework/CHIP/MTROTAProviderDelegateBridge.mm

```diff
@@ -42,6 +42,14 @@
 // TODO Expose a method onto the delegate to make that configurable.
 constexpr uint32_t kMaxBdxBlockSize = 1024;
 constexpr uint32_t kMaxBDXURILen = 256;
+
+// Since the BDX timeout is 5 minutes and we are starting this after query image is available and before the BDX init comes,
+// we just double the timeout to give enough time for the BDX init to come in a reasonable amount of time.
+constexpr System::Clock::Timeout kBdxInitReceivedTimeout = System::Clock::Seconds16(10 * 60);
+
+// Time in seconds after which the requestor should retry calling query image if busy status is receieved
+constexpr uint32_t kDelayedActionTime = 120;
```

Comment:
if this is a delay in seconds, add the unit to the constant name `kDelayedActionTimeSeconds` or similar.

Reply:
done